com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/ShaderPassForward.hlsl

```diff
@@ -178,6 +184,36 @@ void Frag(PackedVaryingsToPS packedInput,
             float4 result = _DebugTransparencyOverdrawWeight * float4(TRANSPARENCY_OVERDRAW_COST, TRANSPARENCY_OVERDRAW_COST, TRANSPARENCY_OVERDRAW_COST, TRANSPARENCY_OVERDRAW_A);
             outColor = result;
         }
+        else if (_DebugFullScreenMode == FULLSCREENDEBUGMODE_QUAD_OVERDRAW)
+        {
```

Comment:
Please reference the original article: https://blog.selfshadow.com/2012/11/12/counting-quads/